app-editors/dav/files/dav-0.9.0-manpage-no-exec.patch

```diff
@@ -0,0 +1,25 @@
+From b8750775a1f38eebe5d49ca1dfff9d6527289d00 Mon Sep 17 00:00:00 2001
```

Comment:
The patch is no longer needed, since you decided to do it according to my suggestion.